docs/deployment/MIGRATIONS.md

```diff
@@ -0,0 +1,32 @@
+# Running commands on staging instances
+
+The following instructions can be used to apply migration commands on an AWS stack of this project.
+
+## Prerequisites
+
+- Access to the AWS Web Console
+
+## Console commands
+
+ - First, log into the [AWS Web Console](https://raster-foundry.signin.aws.amazon.com) and navigate to the [EC2 Container Service](https://console.aws.amazon.com/ecs/home?region=us-east-1#/clusters) dashboard.
+
+ - Go to the [Task Definitions](https://console.aws.amazon.com/ecs/home?region=us-east-1#/taskDefinitions) tab, and look for the migration task called `StagingAppMigrations`. 
+![image](https://cloud.githubusercontent.com/assets/2507188/19808961/1c258a8c-9cf5-11e6-870f-a085a32ba526.png)
```

Comment:
It might be good to introduce some spacing between the previous bullet and this image.